pkgs/racket-test-core/tests/racket/cstruct.rktl

```diff
@@ -156,5 +156,15 @@
   (test #f equal? b (cast b _B-pointer _B-pointer))) ; cast forces new wrapper
 
 ;; ----------------------------------------
+;; Check to ensure offsets are computed correctly
+
+(let ()
+  (test '(0 4 8) compute-offsets (list _int _bool _string))
+  (test '(0 4 5 8) compute-offsets (list _int _byte _byte _long))
```

Comment:
`_long` is not always 8 bytes (not even on 64-bit platforms, thanks to Windows).

Reply:
Good catch, thanks.

Reply:
Looking again, I guess it doesn't matter since the `_long` is at the end, but I like the change to `_int`.